packages/feature-layer/src/feature-layer.ts

```diff
@@ -328,6 +329,30 @@ export function createItemFromTemplate(
   });
 }
 
+export function postProcess(
+  itemId: string,
+  type: string,
+  templates: common.IItemTemplate[],
+  templateDictionary: any,
+  authentication: common.UserSession
+): Promise<any> {
+  return Promise.all([
```

Comment:
@dbouwman per your request, using lower-level utils to fetch the item & data vs `deployUtils._getSolutionTemplateItem`

Reply:
👍